notebooks/06_Serving/06_02_kfserving_sdk.ipynb

```diff
@@ -0,0 +1,256 @@
+    "                            tensorflow=V1alpha2TensorflowSpec(\n",
+    "                              storage_uri='gs://kfserving-samples/models/tensorflow/flowers',\n",
+    "                              resources=V1ResourceRequirements(\n",
+    "                                  requests={'cpu':'100m','memory':'1Gi'},\n",
+    "                                  limits={'cpu':'100m', 'memory':'1Gi'}))))\n",
+    "    \n",
+    "isvc = V1alpha2InferenceService(api_version=api_version,\n",
+    "                          kind=constants.KFSERVING_KIND,\n",
+    "                          metadata=client.V1ObjectMeta(\n",
+    "                              name='flower-sample', namespace=namespace),\n",
+    "                          spec=V1alpha2InferenceServiceSpec(default=default_endpoint_spec))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create InferenceService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Call KFServingClient to create InferenceService."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing = KFServingClient()\n",
+    "KFServing.create(isvc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check the InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.get('flower-sample', namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Canary to InferenceService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Firstly define canary endpoint spec, and then rollout 10% traffic to the canary version, watch the rollout process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "canary_endpoint_spec = V1alpha2EndpointSpec(\n",
+    "                         predictor=V1alpha2PredictorSpec(\n",
+    "                           tensorflow=V1alpha2TensorflowSpec(\n",
+    "                             storage_uri='gs://kfserving-samples/models/tensorflow/flowers-2',\n",
+    "                             resources=V1ResourceRequirements(\n",
+    "                                 requests={'cpu':'100m','memory':'1Gi'},\n",
+    "                                 limits={'cpu':'100m', 'memory':'1Gi'}))))\n",
+    "\n",
+    "KFServing.rollout_canary('flower-sample', canary=canary_endpoint_spec, percent=10,\n",
+    "                         namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rollout more traffic to canary of the InferenceService"
```

Comment:
after this step, user can query service again, for example run 10 times, they will see ~5 from original endpoint, and another 5 from canary.

Reply:
Addressed in the new commit.